Update template files and dependencies

thinkh opened the pull request:
Closes #160

### Summary

Update files using generator-phovea v2.0.1:

- update to papaparse minor version
- update to D3 patch version
- update dev dependencies
- udpate webpack.config.js
- update buildInfo.js

thinkh committed:
Update package-lock.json

oltionchampari reviewed (approved):
Everything works as expected, except circici build.

thinkh committed:
Fix tslint "used before declaration" error

Fixed for some files by reordering the code. For other classes no correct order exists, unless we rewrite or restructure the code. In this case I disabled tslint for the method or file.

oltionchampari reviewed (approved):
Now looks better.
